src/components/layouts/pages/settings/Settings.js

```diff
@@ -20,7 +25,7 @@ function Settings() {
                     <ProfileSelect />
                 </form>
             </section>
-            <PageNavigation back={backButton} />
+            <PageNavigation back={backButton} forward={forwardButton} />
```

Comment:
Let's put this button on `Results` view instead, but I assume you will have to do that anyway during rebase